raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -793,6 +821,7 @@ def secret_learned(
         secrethash,
         payee_address,
         new_payee_state,
+        from_onchain_secretreveal,
```

Comment:
Not the cleanest solution, but duplicating this function wouldn't be better either.